core/domain/services/assets/EspressoEditorAssetManager.php

```diff
@@ -45,7 +45,17 @@ public function addAssets()
      */
     private function registerJavascript()
     {
-        $this->addJs(self::JS_HANDLE_EDITOR, [CoreAssetManager::JS_HANDLE_JS_CORE])->setRequiresTranslation();
+        $dependencies = defined('SCRIPT_DEBUG')
+            ? [
+                ReactAssetManager::JS_HANDLE_REACT_DEV,
+                ReactAssetManager::JS_HANDLE_REACT_DOM_DEV,
+            ]
+            : [
+                ReactAssetManager::JS_HANDLE_REACT_PROD,
+                ReactAssetManager::JS_HANDLE_REACT_DOM_PROD,
+            ];
+        $this->addJs(EspressoEditorAssetManager::JS_HANDLE_EDITOR, $dependencies)
+             ->setRequiresTranslation();
     }
```

Comment:
Did you forget to add `wp-i18n` to deps as explained in the issue? 😄

Reply:
I see that `wp-i18n` is already loaded somewhere for event edit page, can you please check if that's some EDTRv1 js still there?

Reply:
we're calling `->setRequiresTranslation()` which results in the scripts being run through `wp.i18n.setLocaleData()` when loaded...

hmmm... I also saw that `CoreAssetManager` defines `JS_HANDLE_I18N` which I assumed would have `wp-i18n` set as a dependency but that's not actually the case... will have to look into that more